transition-scripts/deploy-gold-standby.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+NAMESPACE=$1
+
+pwd="$(dirname "$0")"
+source "$pwd/helpers.sh"
+
+if ! check_kube_context "api-gold-devops-gov-bc-ca"; then
+    echo "invalid context"
+    exit 1
+fi
+
+
+helm repo add sso-charts https://bcgov.github.io/sso-helm-charts
+helm repo update
+
+cd ./keycloak/
+
+helm upgrade --install sso-keycloak sso-charts/sso-keycloak \
+ -n ${NAMESPACE} -f ./values-gold-${NAMESPACE}.yaml  \
```

Comment:
same as above, I noticed the values-c6af30-dev.yaml is present in the main branch but not `values-gold-c6af30-test.yaml`